### PR TITLE
Create a new test suite in CI pipeline for persistence tests 

### LINKED ISSUE #22058: Create a new test suite in CI pipeline for persistence tests
The task is to create a script to make this new test suite run on hudson.

### PR DESCRIPTION
**Pull request by kokil12**
Fixes #22058 : Adding run_test.sh script to make the "persistence_all" test suite run on hudson.

### CODE REVIEW

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests Please

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests Please

File: appserver/tests/appserv-tests/devtests/persistence/tests/packaging/resultCount.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+echo "------------  ========="
+echo "FAILED=   $FAILED"
+echo "------------  ========="
+echo "DID NOT RUN=   $DNR"
+echo "------------  ========="
+echo "Total Expected=$TOTAL"
+echo "************************"
+echo ""
+
+echo "************************">>$APS_HOME/devtests/cmp/count.txt;
```
Comment (yaminikb) [inline appserver/tests/appserv-tests/devtests/persistence/tests/packaging/resultCount.sh]: Fix the count.txt path in all lines, it is incorrectly pointing to "cmp"

**Review by yaminikb — CHANGES_REQUESTED** [review]

**Review by yaminikb — DISMISSED** [review]

File: appserver/tests/appserv-tests/devtests/persistence/run_test.sh [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  $S1AS_HOME/bin/asadmin stop-domain   
+}
+
+run_test_id(){
+  #a common util script located at main/appserver/tests/common_test.sh
+  source `dirname $0`/../../../common_test.sh
+  kill_process
+  delete_gf
+  download_test_resources glassfish.zip version-info.txt
+  unzip_test_resources $WORKSPACE/bundles/glassfish.zip
```
Comment (arindam-bandyopadhyay) [inline appserver/tests/appserv-tests/devtests/persistence/run_test.sh]: Please add the cleanup logic. Please refer https://github.com/javaee/glassfish/blob/master/appserver/tests/appserv-tests/devtests/admin/cli/run_test.sh#L72

**Review by arindam-bandyopadhyay — CHANGES_REQUESTED** [review]

**Comment by arindam-bandyopadhyay** [comment]
Please add the test id in appserver/tests/test_groups/gating

**Comment by arindam-bandyopadhyay** [comment]
@glassfishrobot Run CI tests Please

**Comment by kokil12** [comment]
@glassfishrobot Run CI tests Please

**Review by arindam-bandyopadhyay — APPROVED** [review]

**Pull request merged**